DEVELOPER.rst

```diff
@@ -1,71 +1,143 @@
-=================
- Install package
-=================
+Developer guide
+###############
 
-Install in develop mode to notice changes immediately.
+This document should guide you through the setup of your local developer
+environment, as well as through the release process of a new version.
 
-.. code::
+====================================
+Setting up a development environment
+====================================
 
-   python setup.py develop
+Download code
+-------------
 
-=================
- Doing a Release
-=================
+First clone the repository to your local machine::
 
-Update version
---------------
+    $ git clone https://github.com/bio-phys/MDBenchmark.git
 
-Change version info in `mdbenchmark/__init__.py`
+Create a virtual environment
+----------------------------
 
-Updating CHANGELOG
-------------------
+If you are using ``conda``, you can easily create a `virtual environment`_::
 
-Install and run towncrier in the project directory.
+    $ conda create -n benchmark python=3
```

Comment:
`conda create -n benchmark  mdsynthesis click` 

This should install all dependencies. I don't care much which python version people use. The CI server tests all anyway.

In your current setup, the dependencies will be pulled from pip.